slick-testkit/src/codegen/scala/slick/test/codegen/GenerateMainSources.scala

```diff
@@ -143,13 +143,16 @@ val  SimpleA = CustomTyping.SimpleA
           |  import java.util.UUID
           |  val u1 = UUID.randomUUID()
           |  val u2 = UUID.randomUUID()
-          |  val p1 = PersonRow(1, u1)
-          |  val p2 = PersonRow(2, u2)
+          |  val u3 = UUID.randomUUID()
+          |  val u4 = UUID.randomUUID()
+          |  val p1 = PersonRow(1, u1, uuidFunc = Some(u3))
+          |  val p2 = PersonRow(2, u2, uuidFunc = Some(u4))
```

Comment:
According to the generated `CG10.scala`, `uuidFunc` should be `uuidDef`.

_@cvogt seems the case wasn't run as we expected._